Stripe/STPPaymentMethod.m

```diff
@@ -29,23 +29,50 @@ @interface STPPaymentMethod ()
 
 @implementation STPPaymentMethod
 
+- (NSString *)description {
+    NSArray *props = @[
+                       // Object
+                       [NSString stringWithFormat:@"%@: %p", NSStringFromClass([self class]), self],
+                       
+                       // Identifier
+                       [NSString stringWithFormat:@"stripeId = %@", self.stripeId],
+                       
+                       // STPPaymentMethod details (alphabetical)
```

Comment:
These aren't actually alphabetical. It's probably worth alphabetizing them because I think it makes it easier to consume the logged value, but you could also just update the comment.